appconfig/settings/wdb.q

```diff
@@ -0,0 +1,5 @@
+// Bespoke WDB settings for AWS finspace Starter Pack
+\d .wdb
+replay:0b;    //set replay to false so that replay isnt carried out during intial subscribe process
+upd:(::);    //set upd to discard data until the end of period signals the start of new period
```

Comment:
no functional difference, but to me it seems a bit clearer to set this as a function (that does nothing)?

```suggestion
upd:{[t;x]};    //set upd to discard data until the end of period signals the start of new period
```